python/pyspark/sql/tests/test_catalog.py

```diff
@@ -200,6 +200,18 @@ def test_list_columns(self):
                     "does_not_exist",
                     lambda: spark.catalog.listColumns("does_not_exist"))
 
+    def test_table_exists(self):
+        spark = self.spark
+        with self.database("some_db"):
+            spark.sql("CREATE DATABASE some_db")
+            with self.table("tab1", "some_db.tab2"):
+                self.assertFalse(spark.catalog.tableExists("tab1"))
+                self.assertFalse(spark.catalog.tableExists("tab2", "some_db"), [])
```

Comment:
```suggestion
                self.assertFalse(spark.catalog.tableExists("tab2", "some_db"))
```
Mistake?

Reply:
Yes, sorry, I had copied some of the initial setup and then forgot to remove the empty array